app/src/main/kotlin/com/intive/picover/parties/viewmodel/PartiesViewModel.kt

```diff
@@ -19,9 +22,17 @@ import kotlinx.coroutines.launch
 @HiltViewModel
 class PartiesViewModel @Inject constructor(
 	private val partiesRepository: PartiesRepository,
-) : ViewModel() {
+	@Validator.ShortText private val shortTextValidator: TextValidator,
+	@Validator.LongText private val longTextValidator: TextValidator,
+
+	) : ViewModel() {
 
 	val state: MutableState<ViewModelState<List<Party>>> = mutableStateOf(Loading)
+	private var _title = mutableStateOf("")
```

Comment:
shouldn't this be a val, as it is a mutable state ? is there a need to declare this as a var ?